.ci/install_kata.sh

```diff
@@ -17,6 +17,7 @@ KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 experimental_qemu="${experimental_qemu:-false}"
 TEST_RUST_AGENT="${TEST_RUST_AGENT:-false}"
 TEST_CGROUPSV2="${TEST_CGROUPSV2:-false}"
+RUN_KATA_CHECK="${RUN_KATA_CHECK:-true}"
```

Comment:
One more nit, could you add this variable here as well, https://github.com/kata-containers/tests/blob/master/.ci/jenkins_job_build.sh#L11